server/game/cards/03_TWI/events/PetitionTheSenate.ts

```diff
@@ -0,0 +1,25 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { EventCard } from '../../../core/card/EventCard';
+import { Trait } from '../../../core/Constants';
+
+export default class PetitionTheSenate extends EventCard {
+    protected override getImplementationId() {
+        return {
+            id: '3500129784',
+            internalName: 'petition-the-senate',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.setEventAbility({
+            title: 'If you control 3 or more Official units, draw 3 cards.',
```

Comment:
We don't include any "if" conditions in the title because if the player is seeing this string, the condition is already met. So we can just say `Draw 3 cards`. Same comment for other titles with "if" in the name in this PR